lib/identity/stateTransition/IdentityTopUpTransition/validation/state/validateIdentityTopUpTransitionStateFactory.js

```diff
@@ -0,0 +1,20 @@
+const ValidationResult = require('../../../../../validation/ValidationResult');
+
+/**
+ * @return {validateIdentityTopUpTransitionState}
+ */
+function validateIdentityTopUpTransitionStateFactory() {
+  /**
+   * @typedef {validateIdentityTopUpTransitionState}
+   * @param {IdentityTopUpTransition} stateTransition
+   * @return {Promise<ValidationResult>}
+   */
+  // eslint-disable-next-line no-unused-vars
+  async function validateIdentityTopUpTransitionState(stateTransition) {
```

Comment:
Why do we need to keep `stateTransition` here?

Reply:
Just to keep interface since we still passing the argument here.